src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -2017,6 +2017,36 @@ string[] linkingTargetFields
             }
         }
 
+        /// <summary>
+        /// This test checks that the final config used by runtime engine doesn't lose the directory information
+        /// provided in the base config file path when loading the config based on the environment.
+        /// </summary>
+        /// <param name="addDirectory"></param>
+        /// <param name="environmentValue"></param>
+        /// <param name="baseConfigFilePath"></param>
+        /// <param name="finalConfigFilePath"></param>
+        [DataTestMethod]
+        [DataRow(false, "", "current-dir-dab-config.json", "current-dir-dab-config.json", DisplayName = "Config file in the current directory")]
+        [DataRow(true, "", "test\\diff-dir-dab-config.json", "test\\diff-dir-dab-config.json", DisplayName = "Config file in a different directory")]
```

Comment:
What if its an absolute path to the config file? for example, `C:\\dab\\test\\diff-dir-dab-config.json` ?